crates/ruff_python_formatter/src/core/helpers.rs

```diff
@@ -48,8 +47,8 @@ pub fn expand_indented_block(
     locator: &Locator,
 ) -> (Location, Location) {
     let contents = locator.contents();
-    let start_index = locator.index(location);
-    let end_index = locator.index(end_location);
+    let start_index = locator.offset(location);
```

Comment:
`index` got renamed to `offset` (`index` is also a field on the `Locator` struct -- this method didn't exist at all in the "upstream" version of `Locator` from `ruff_python_ast`).